### PR TITLE
Deprecate "Detached Elements" tool

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article for review: 
* **Debug DOM memory leaks with the Detached Elements tool**
   * https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks?branch=pr-en-us-3267
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks)

AB#53967018

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md [hunk 1]
```diff
@@ -6,10 +6,23 @@ ms.author: msedgedevrel
...
+ms.date: 09/16/2023
 ---
 # Debug DOM memory leaks with the Detached Elements tool
 
+> [!IMPORTANT]
+> The **Detached Elements** tool is being deprecated and eventually removed.  The same functionality is now available in the **Memory** tool instead.  Deprecation period:
+>   * Microsoft Edge 130: The **Detached Elements** tool is still available, but the **Memory** tool is recommended instead.
+>   * Microsoft Edge 133: The **Detached Elements** tool is removed.
+>
+> The downstream-only CDP function `EdgeDOMMemory.getDetachedNodesIds` is deprecated; for the moment, it still works, but use `DOM.getDetachedDomNodes` instead.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md:18]: is this function deprecated in same sense as DE tool & same timeline?  @captainbrosset

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md:18]: chrome devtools protocol (cdp)

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md]: greatly clarified & spelled out specifics

File: microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md [hunk 2]
```diff
@@ -6,10 +6,21 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: devtools
-ms.date: 07/05/2023
+ms.date: 09/18/2023
 ---
 # Debug DOM memory leaks with the Detached Elements tool
 
+> [!IMPORTANT]
+> The **Detached Elements** tool is being deprecated.  Starting with Microsoft Edge 130, the **Detached Elements** tool has a message stating that the tool is deprecated; instead, in the **Memory** tool, in the initial **Select profiling type** screen, select the **Detached elements** option button.  The downstream-only Chrome DevTools Protocol (CDP) function `EdgeDOMMemory.getDetachedNodesIds` continues to work, but use `DOM.getDetachedDomNodes` instead.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md:14]:
```suggestion
> The **Detached Elements** tool is being deprecated.  Starting with Microsoft Edge 130, the **Detached Elements** tool has a message stating that the tool is deprecated; instead, in the **Memory** tool, in the initial **Select profiling type** screen, select the **Detached elements** option button.  The Edge-only Chrome DevTools Protocol (CDP) function `EdgeDOMMemory.getDetachedNodesIds` continues to work, but use `DOM.getDetachedDomNodes` instead.
```

**Review by captainbrosset — APPROVED** [review]
Approve with a small change request.

**Pull request merged**